src/linux/py_proc.h

```diff
@@ -68,6 +68,21 @@ union {
 
 
 // ----------------------------------------------------------------------------
+static Elf64_Addr
+_py_proc_get_base64(Elf64_Ehdr * ehdr, void * elf_map)
+{
+  Elf64_Addr res = (Elf64_Addr) 0 - 1;
+  for (uint16_t i = 0; i < ehdr->e_phnum; ++i) {
+    Elf64_Phdr * phdr = (Elf64_Phdr *) (elf_map + i * ehdr->e_phoff);
+    if (phdr->p_type == PT_LOAD) {
+      Elf64_Addr value = phdr->p_vaddr - phdr->p_vaddr % phdr->p_align;
+      if (value < res)
+        res = value;
+    }
+  }
```

Comment:
I imagine that, once we have found the offset, we can break off the loop and return. Please consider adding a `break` here.

Reply:
Yes.